tardis/model/base.py

```diff
@@ -189,6 +196,33 @@ def abundance(self):
         abundance.columns = range(len(abundance.columns))
         return abundance
 
+    def decay(self, time_explosion, normalize=True):
+
+        if not self.decayed:
+            self.decayed = True
+        else:
+            raise ValueError("Isotopic Abundance already decayed")
+
+        isotope_abundance = self.raw_isotope_abundance.decay(time_explosion)
+
+        #Set atomic_number as index in isotopic_abundance dataframe
+        isotope_abundance = isotope_abundance.reset_index().drop(
+            'mass_number', axis=1).set_index(['atomic_number'])
+
+        #Merge abundance dataframes
+        modified_df = isotope_abundance.append(self.abundance)
```

Comment:
@wkerzendorf I have now used `pd.concat()` in the latest commit . 
- If you mean `pd.add()` , then according to what I could find , `index values` for both `Dataframe` must be equal, then only result is correct, otherwise, values not present in one dataframe , are filled with `NaN` values
- `pd.merge()` and `pd.join()` operation results in addition along columns.
So, I think , it leaves me with `pd.concat()` , and I have also added a test to ensure if it works correctly.